### ISSUE TITLE
Ignore %ext%  when generating %num%

### ISSUE THREAD

**Issue by DevYama** [Bionus/imgbrd-grabber#1524]
**Is your feature request related to a problem? Please describe**
Sorta, if a filenaming is %character%-%num%, it would name files like character-1.png, character-2.png and so on and so on, but if the file is different ext (jpg for an example) it would be named character-1.jpg. 

**Describe the solution you'd like**
A setting to ignore file extension when naming files.

**Describe alternatives you've considered**
Set preferred file extension 

**Additional context**
Haven't checked the outcome of filenaming when batch downloading.

**Comment by Bionus** [comment]
So from what I understand, you want a setting to make the extension optional in the filename? And Grabber would then automatically add the `.%ext%` at the end by itself?

I understand where you're coming from, however for multiple reasons this feature should be disabled by default, and isn't it easier to simply add the extension that going to the settings to enable this "automatic extension" feature?

**Comment by DevYama** [comment]
No, let's say we have filenaming like ```%character%-%num%.%ext%```, this would simply name it like ```charactername-1.png``` or ```charactername-1.jpg```, the problem is with ```%num%``` and how it handles ```%ext%```.
Let's say we have the previous ```%character%-%num%.%ext%```, but the images format goes from png to jpg, resulting in the ```%num%``` going back to 1, for jpg images, I wish it would keep going with the same numbering for different file extensions. 

tl;dr
```%character%-%num%.%ext%``` results in ```%num%``` being annoying. (Notice ```%num%``` and ```%ext%```)
![filenaming](https://user-images.githubusercontent.com/26037681/49791083-5ae8c400-fd38-11e8-8213-60b2fcfdd45b.png)

**Comment by Bionus** [comment]
In that case I understand.

I'm working on this and will add some `%num:noext%` option or something, to be able to make it ignore the file's extension. 👍
